### PR TITLE
docs issue # 47

### PR DESCRIPTION
**Pull request by bschumac**
In response to [#47](https://github.com/openEOPlatform/documentation/issues/47)

### CODE REVIEW

**Comment by bschumac** [comment]
@m-mohr any thoughts? Please go ahead and directly merge this if there are no further questions/issues.

**Comment by bschumac** [comment]
@m-mohr can this be merged?

**Comment by m-mohr** [comment]
Have you checked that it renders correctly?
If yet, go ahead an merge it. 
You still have to get rid of the merge conflicts though.

File: getting-started/jupyterlab/index.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ It has the openEO Python client pre-installed, but it does not support running t
 
 You need to authenticate before you can use it:
 
-1. Click the "Sign in with EODC Identity Providers" button
-2. Now you need to select the "EGI" button on the right (instead of directly typing in your credentials on the left). It will start the EGI Authentication workflow for openEO Platform. For details check the documentation to join the [Free Tier](../../join/free_tier.md) or the [Early Adopter program](../../join/early_adopter.md).
+1. Select from the "Sign in " dropdown menu the "openEO Platform" option
```
Comment (m-mohr) [inline getting-started/jupyterlab/index.md:13]:
```suggestion
1. Select from the "Sign in " dropdown menu the "openEO Platform" option
```

The character after the "Sign in" is not displayed correctly for me.

Comment (m-mohr) [reply getting-started/jupyterlab/index.md:13]:
```suggestion
1. Select from the "Sign in" dropdown menu the "openEO Platform" option
```

**Pull request merged**